db/migrate/20990101999999_sample_migration.foreman_plugin_template.rb

```diff
@@ -0,0 +1,2 @@
+class SampleMigration < ActiveRecord::Migration
```

Comment:
Where can I give examples to this? Do we have to rely only on docs for it?

My long-term suggestion would be a generator, but I don't think it's the right time to do that.